sigstore/_verify.py

```diff
@@ -144,6 +139,15 @@ def verify(
         success.
         """
 
+        # NOTE: The `X509Store` object currently cannot have its time reset once the `set_time`
+        # method been called on it. To get around this, we construct a new one for every `verify`
+        # call.
+        store = X509Store()
+        for parent_cert_pem in self._fulcio_certificate_chain:
+            parent_cert = load_pem_x509_certificate(parent_cert_pem)
+            parent_cert_ossl = X509.from_cryptography(parent_cert)
```

Comment:
Actually it'd probably be better to do this in the constructor and just store the `parent_cert_ossl`s for each chain member.